MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Poll/PollOptionRow.swift

```diff
@@ -38,12 +43,55 @@ public struct PollOptionRow: View {
                     viewModel.shouldBecomeFirstResponder = false
                     viewModel.textField?.becomeFirstResponder()
                 }
+
+                if #available(iOS 16.0, *) {
+                    field.accessibilityActions {
+                        if let moveUp {
+                            Button(L10n.Scene.Compose.Poll.moveUp, action: moveUp)
+                        }
+                        if let moveDown {
+                            Button(L10n.Scene.Compose.Poll.moveDown, action: moveDown)
+                        }
+                        if let removeOption {
+                            Button(L10n.Scene.Compose.Poll.removeOption, action: removeOption)
+                        }
+                    }
+                } else {
+                    switch (moveUp, moveDown, removeOption) {
```

Comment:
@j-f1 I've used switch pattern matching here and I think it's fine. Thanks for your help!

Reply:
That looks good! And when we drop iOS 15 support the whole switch can be removed.

Reply:
The good thing is also that you basically don'T forget to cover cases as the compiler would otherwise complain about the switch statement.